cypress/e2e/hermioneAccount.spec.js

```diff
@@ -1,11 +1,98 @@
 /// <reference types='cypress' />
+import { faker } from '@faker-js/faker';
 
 describe('Bank app', () => {
-  before(() => {
-    cy.visit('/');
+  beforeEach(() => {
+    cy.visit('');
   });
 
+  const userHermiona = 'Hermoine Granger';
+  const accounNumber = 1001;
+  const currency = 'Dollar';
+  const amountPlaceholder = 'amount';
+  const depositAmount = `${faker.number.int({ min: 1648, max: 2023 })}`;
+  const buttonDeposit = 'Deposit';
+  const depositMessage = 'Deposit Successful';
+  const buttonWithdrawl = 'Withdrawl';
+  const withdrawlAmount = `${faker.number.int({ min: 1648, max: depositAmount })}`;
```

Comment:
The variable name `withdrawlAmount` seems to have a typo. It should likely be `withdrawalAmount`.